openmetadata-service/src/main/java/org/openmetadata/service/apps/bundles/changeEvent/generic/GenericPublisher.java

```diff
@@ -90,13 +90,14 @@ public void sendMessage(ChangeEvent event) throws EventPublisherException {
   public void sendTestMessage() throws EventPublisherException {
     long attemptTime = System.currentTimeMillis();
     try {
-      prepareAndSendMessage(TEST_MESSAGE_JSON, getTarget());
+      prepareAndSendMessage(TEST_MESSAGE_JSON, getTarget(), true);
     } catch (Exception ex) {
       handleException(attemptTime, ex);
     }
   }
 
-  private void prepareAndSendMessage(String json, Invocation.Builder target) {
```

Comment:
@Siddhanttimeline again here we are passing test destination flag , I feel we should write different method , and under that reusable methods should be created rather than relating flag from top calls

Reply:
Noted.